### PR TITLE
📝 development setup instructions

### PR DESCRIPTION
**Pull request by m0ppers**
not sure about best practices concerning the rails stuff but this works for me (tm)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,23 +42,33 @@ When a guest checks in, a random ID will be saved on the guest's device. **recov
...
 
-**Before you can start, you need:**
+It is also advisable to install the gems locally and not system wide:
 
-- ...
-- ...
-- ...
+```
+export GEM_HOME=$HOME/.gem
+export PATH=$PATH:$HOME.gem/bin
```
Comment (salimhb) [inline README.md:60]: This depends on your own ruby setup. I wouldn't include this here as it's not specific for this app. Example, if you use [rvm](http://rvm.io/), this is already done.

File: README.md [hunk 2]
```diff
@@ -42,23 +42,33 @@ When a guest checks in, a random ID will be saved on the guest's device. **recov
...
+```
 
-**Setup:**
+Then execute:
 
-You can use the provided docker-compose file to locally run redis and postgres, but it is not required.
```
Comment (salimhb) [inline README.md:61]: Why remove this notice?
It's a matter of taste what each dev prefers to use. Some don't use docker and manage their own installation of postgres/redis.

File: README.md [hunk 3]
```diff
@@ -42,23 +42,33 @@ When a guest checks in, a random ID will be saved on the guest's device. **recov
...
+```
+bundler install
+echo "DATABASE_URL=postgres://postgres:mysecretpassword@127.0.0.1:5432/" >> .env.local
+docker-compose up -d # will start database in the background
+rails db:migrate RAILS_ENV=development
```
Comment (salimhb) [inline README.md:69]:
```suggestion
rails db:setup RAILS_ENV=development
```

Comment (salimhb) [reply README.md:69]: This would create the database and run migrations.

**Comment by m0ppers** [comment]
superseded by #49

**Pull request closed without merging**